### PR TITLE
relase and documentation work

### PR DESCRIPTION
**Pull request by VarunSharma15**
### Description

This PR is to add Changelog and documentation changes.

### Checklist
 - [ ] I have run this code in development and it appears to resolve the stated issue
 - [ ] This PR includes tests, or tests are not required/relevant for this PR
 - [ ] I have updated the `CHANGELOG.md` with information about my change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ This [dbt](https://github.com/dbt-labs/dbt) package provides compatibility with
 | Schema tests          | sequential_values             | :white_check_mark:    | no customization needed                                                |
 | Schema tests          | unique_combination_of_columns | :white_check_mark:    | no customization needed                                                |
 | Schema tests          | accepted_range                | :white_check_mark:    | no customization needed                                                |
+| Schema tests          | at_least_one                | :white_check_mark:    |  custom macro provided                                               |
+| Schema tests          | expression_is_true             | :white_check_mark:    |  custom macro provided                                               |
```
Comment (tallamohan) [inline README.md:65]: Correct indentation

File: README.md [hunk 2]
```diff
@@ -71,10 +73,11 @@ This [dbt](https://github.com/dbt-labs/dbt) package provides compatibility with
 | SQL generators        | star                          | :white_check_mark:    | no customization needed                                                |
 | SQL generators        | union_relations               | :white_check_mark:    | custom macro provided                                                  |
 | SQL generators        | generate_series               | :white_check_mark:    | custom macro provided                                                  |
-| SQL generators        | surrogate_key                 | :white_check_mark:    | custom macro provided, [additional install steps required](#additional-steps-for-surrogate_key-macro)               |
+| SQL generators        | generate_surrogate_key             | :white_check_mark:    | custom macro provided, [additional install steps required](#additional-steps-for-surrogate_key-macro)               |
```
Comment (tallamohan) [inline README.md:76]: Correct indentation

File: README.md [hunk 3]
```diff
@@ -71,10 +73,11 @@ This [dbt](https://github.com/dbt-labs/dbt) package provides compatibility with
...
 | SQL generators        | safe_add                      | :white_check_mark:    | no customization needed                                                |
 | SQL generators        | pivot                         | :white_check_mark:    | no customization needed                                                |
 | SQL generators        | unpivot                       | :white_check_mark:    | no customization needed, see [compatibility note](#unpivot)            |
+| SQL generators        | deduplicate                        | :white_check_mark:    | custom macro provided                                                |
```
Comment (tallamohan) [inline README.md:80]: Correct indentation

File: README.md (hunk 1 above)
Comment (VarunSharma15) [reply README.md:65]: These will be converted into table, indentation won't matter much, Mohan.

File: README.md (hunk 2 above)
Comment (VarunSharma15) [reply README.md:76]: These will be converted into table, indentation won't matter much, Mohan.

File: README.md (hunk 3 above)
Comment (VarunSharma15) [reply README.md:80]: These will be converted into table, indentation won't matter much, Mohan.

**Review by tallamohan — APPROVED** [review]
Looks Good

**Review by sc250072 — APPROVED** [review]
Looks Good

**Pull request merged**